extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "Enables users to develop and publish database schemas for MSSQL Databases",
-  "version": "0.14.0",
+  "version": "0.15.0",
```

Comment:
the version was already bumped to 0.14.0 after the October release, so this doesn't need to be bumped again. The current version in vscode is 0.13.1

Reply:
Will revert the version, thanks!